docs/faq/templates.md

```diff
@@ -22,27 +22,27 @@
 		- https://blog.melbadry9.xyz/dangling-dns/aws/ddns-ec2-current-state
 		- https://blog.projectdiscovery.io/writing-nuclei-templates-for-wordpress-cves/
 
-??? info "How to run nuclei templates?"
+??? info "How do I run nuclei templates?"
 
-	Nuclei templates can be executed using template name or tags, using `-t`, `-tags` flag, respectively.
+	Nuclei templates can be executed using a template name or with tags, using `-templates` (`-t`) and `-tags` flag, respectively.
 
 	```
```

Comment:
We could add "shell", "sh" or "bash" tag after the backticks

Reply:
I don't believe this is supported for inline code, only for blocks. I've opened https://github.com/projectdiscovery/nuclei-docs/issues/46 to update all the code blocks.